gpytorch/variational/variational_strategy.py

```diff
@@ -85,24 +85,36 @@ def forward(self, x):
             :obj:`gpytorch.distributions.MultivariateNormal`: The distribution q(f|x)
         """
         variational_dist = self.variational_distribution.variational_distribution
+
         if torch.equal(x, self.inducing_points):
             return variational_dist
         else:
             n_induc = self.inducing_points.size(-2)
-            full_inputs = torch.cat([self.inducing_points, x])
+            full_inputs = torch.cat([self.inducing_points, x], dim=-2)
             full_output = self.model.forward(full_inputs)
             full_mean, full_covar = full_output.mean, full_output.lazy_covariance_matrix
 
-            test_mean = full_mean[n_induc:]
-            induc_mean = full_mean[:n_induc]
-            induc_induc_covar = full_covar[:n_induc, :n_induc].add_jitter()
-            induc_data_covar = full_covar[:n_induc, n_induc:]
-            data_induc_covar = full_covar[n_induc:, :n_induc]
-            data_data_covar = full_covar[n_induc:, n_induc:]
+            if full_covar.dim() == 3:
```

Comment:
Eventually, we can collapse this using ellipses. However, ATM I don't think that our __getitem__ supports ellipses, so that's for a later date.